cpp/roaring64map.hh

```diff
@@ -345,11 +381,14 @@ public:
      */
     Roaring64Map &operator|=(const Roaring64Map &r) {
         for (const auto &map_entry : r.roarings) {
-            if (roarings.count(map_entry.first) == 0) {
-                roarings[map_entry.first] = map_entry.second;
-                roarings[map_entry.first].setCopyOnWrite(copyOnWrite);
-            } else
-                roarings[map_entry.first] |= map_entry.second;
+            auto result = roarings.insert(map_entry);
+            if (result.second) {
+                // Key not present in destination, so insert was performed.
+                result.first->second.setCopyOnWrite(copyOnWrite);
```

Comment:
Nit: Could we assign `result.first->second` to a variable to clarify what it is? Same elsewhere.

Reply:
Done.